src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class FigureSupplier {
+
+    public static Figure getRandomFigure() {
+        int randomNumber = (int) (Math.random() * 5);

```

Comment:
Use `Random` instance (constant), any limiters should be extracted as a constants.

Reply:
Not fixed